nifi-commons/nifi-security-utils-api/src/test/groovy/org/apache/nifi/security/util/TlsConfigurationTest.groovy

```diff
@@ -1,90 +0,0 @@
-/*
```

Comment:
Why are you removing a bunch of unit tests without Java replacements in this PR? Did I miss that scoped into the ticket?

Reply:
Thanks for the feedback, I put a short explanation for some items in the pull request description, but it is admittedly not described in the Jira issue, so I can certainly adjust the scope one way or the other.

Most of the removals are related directly to JUnit 4 or Groovy Test dependencies, where the tests themselves did not seem worth migrating. If there are some that you think are worth keeping, I can make the adjustments accordingly. To categorize the removals:

1. TLS Configuration and Socket Tests: These tests were limited to checking available TLS versions, primarily for earlier versions of Java 8. With Java 11 as the baseline for the main branch, these tests no longer seem to provide useful checks.

2. Record Utility Tests: These tests were very narrowly scoped with limited methods, and some of the functionality is already covered in other tests

3. Kudu Integration Tests: These tests cover basic uses, but as integration tests, they are not run regularly. [NIFI-11138](https://issues.apache.org/jira/browse/NIFI-11138) outlines potential migration to Testcontainers. These tests require JUnit 4 and since they are narrowly focused, it seems better to rewrite them from scratch using Testcontainers versus building off the existing approach.

4. Registry Properties Tests: With refactoring several versions back, the runtime components are already tested in shared properties classes, so these Registry-specific tests do not add much coverage and even if they were migrated, they should be scaled down significantly.

5. Registry-specific encrypt-config Toolkit Tests: These are Groovy-based tests that also have a lot of overlap with shared encrypt-config code, and shared flow encryption and property encryption modules in nifi-commons.

With that background, I'm open to making adjustments to keep certain tests if you think there is particular value in some cases. I will also update the Jira description to note that some tests should be removed as part of this effort.